### PR TITLE
Autocomplete Topo yaml files

### PR DESCRIPTION
**Pull request by kellerza**

### CODE REVIEW

**Comment by hellt** [comment]
ah, so that is why when I hit `tab` after `deploy --topo` I didn't get a list of files?
where can I read about it more?

File: docs/cmd/completion.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ To load completions for each session, execute once:
 ```bash
 containerlab completion zsh > "${fpath[1]}/_containerlab"
 ```
+> Note: `$fpath[1]` in this command refers to the first path in `$fpath`. Ensure you use
```
Comment (hellt) [inline docs/cmd/completion.md:36]: can you change the blockquote style here to a theme supported info block that is enabled like this:

```
!!!info
    your text here
```

**Review by hellt — CHANGES_REQUESTED** [review]

File: cmd/completion.go [hunk 2]
```diff
@@ -40,19 +40,12 @@ fish:
...
-
-  # To load completions for every new session, run:
-  PS> containerlab completion powershell > containerlab.ps1
-  # and source this file from your PowerShell profile.
 `,
 	DisableFlagsInUseLine: true,
 	ValidArgs:             []string{"bash", "zsh", "fish"},
 	Args:                  cobra.ExactValidArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
+		cmd.Root().MarkFlagFilename("topo", "*.yaml", "*.yml")
```
Comment (hellt) [inline cmd/completion.go:48]: I also think that it is best to add this line not in the completions file, but in the file where `topo` flag is defined

Comment (kellerza) [reply cmd/completion.go]: ok, moved back to root.go

**Pull request merged**